content/docs/reference/cli-me.md

```diff
@@ -22,7 +22,11 @@ neonctl me
 
 ### Options
 
-Only [global options](/docs/reference/neon-cli#global-options) apply.
+In addition to the Neon CLI [global options](/docs/reference/neon-cli#global-options), the `me` subcommand supports this option:
+
```

Comment:
Same here, context is only for project/branch scoped commands

Reply:
Fixed.